packages/flutter/lib/src/material/input.dart

```diff
@@ -47,6 +47,7 @@ class InputField extends StatefulWidget {
     this.keyboardType: TextInputType.text,
     this.hintText,
     this.style,
+    this.hintStyle,
```

Comment:
We should probably plumb this out through all the various text input widgets.  Also, we need a test.  :)